test/index.spec.ts

```diff
@@ -123,7 +127,7 @@ const getRuleName = (configuredRule: string): string => {
 };
 
 /*
- * Tests
+ * Tests.
  */
```

Comment:
Wait... would this be reported as an error?

Reply:
yes, unless you configure it to not do it. (which can be done)

Reply:
It is used a lot at Dott, in many repositories. Is it wise to ban it by default?

Reply:
same case for no magic number rule, it is disabled for tests